recipes/jruby/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "jruby" %}
+{% set version = "9.1.13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+
+source:
+  url: https://s3.amazonaws.com/jruby.org/downloads/9.1.13.0/jruby-bin-9.1.13.0.tar.gz
+  sha256: 9d156646623ac2f27174721035b52572a4b05690db7c1293295aa2c04aad3908
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
You need to have a build script to compile jruby. Otherwise this is just a package with no files.

Reply:
Ah, that's why the last line often say "Uploading no packages " ?

But for jruby / groovy etc we only really need to have a "brew install" like arrangement with the "openjdk" being a pre-requisite i.e. we only need to have the binary file within the Conda env.

This is quite similar to rust-feedstock or go-feedstock which don't build anything but place the proper binaries in the environment.

If this is not the right direction could you please help me out with this then ?